core/render/html/default.py

```diff
@@ -310,252 +308,215 @@ def renderBoneValue(self,
 
         return None
 
-    def add(self, skel: SkeletonInstance, tpl: str = None, params: Any = None, *args, **kwargs) -> str:
+    def get_action_template(self, action: str, tpl: str) -> Template:
```

Comment:
I think `template` would be a more expressive parameter-name than `tpl`.
In this method this shouldn't be a problem. In the others (`view`, etc.) we would have a backward compatibility problem – But would be nice as well.